ras_backstage/controllers/party_controller.py

```diff
@@ -62,3 +62,20 @@ def get_businesses_by_search(query):
 
     logger.debug('Successfully retrieved businesses by search query')
     return json.loads(response.text)
+
+
+def update_respondent_details(respondent_id, first_name, last_name, telephone):
+    logger.debug('Updating respondent details', respondent_id=respondent_id)
+    url = f'{app.config["RAS_PARTY_SERVICE"]}party-api/v1/respondents/change_respondent_details'
+    payload = {
+        "respondent_id": respondent_id,
+        "firstName": first_name,
+        "lastName": last_name,
+        "telephone": telephone
+    }
+
+    response = request_handler('PUT', url, json=payload, auth=app.config['BASIC_AUTH'])
+
+    if response.status_code != 200:
+        logger.error('Error updating respondent details', respondent_id=respondent_id)
+        raise ApiError(url, response.status_code)
```

Comment:
We should have a debug level success log at the end of this function to be consistent with the rest of the app